### ISSUE TITLE
None of the entities are working with upgrade to v1.0.42-RC2

### ISSUE THREAD

**Issue by bslefevre** [lendy007/homeassistant-skodaconnect#43]
**Describe the bug**
None of the entities are working, no errors are being shown (see screenshot)
![image](https://user-images.githubusercontent.com/5653833/124121865-b5240e80-da75-11eb-8f69-657bc17ffc26.png)

**Latest working release**
v1.0.40-RC5

**Additional context**
I've downgraded back to RC5 and now it works again
![image](https://user-images.githubusercontent.com/5653833/124122009-da188180-da75-11eb-867e-68585c501cfa.png)

**Comment by derjungewilde** [comment]
I have the same issue. but my last working version is v.1.0.40-RC2


![Übersicht_-_Home_Assistant](https://user-images.githubusercontent.com/4105139/124124004-37153700-da78-11eb-84c5-9fccec2cf0ac.jpg)
![Übersicht_-_Home_Assistant-2](https://user-images.githubusercontent.com/4105139/124124012-38defa80-da78-11eb-8548-5de9bac4f7f5.jpg)

**Comment by jhqv** [comment]
I am also experiencing issues. After updating to rc1 and then rc2 all entities disappeared. After figuring out how to run the config wizard I get stuck at the VIN selection which doesn't list my vehicle, despite the correct Skoda connect credentials. I don't see anything relevant in the logs, unfortunately.

I also tried removing the integration from HACS, rebooting and reinstalling it but the VIN is still not visible.

**Comment by derjungewilde** [comment]
> @bslefevre @derjungewilde did it work at all after setup?
> Did you go through the config after updating?
> What does it say under integrations i GUI?
> ![Screenshot_20210701-151408_Home Assistant](https://user-images.githubusercontent.com/1235384/124130536-2b793e80-da7f-11eb-8066-2481b1d75718.jpg)

I don't see this Integration in the GUI after installing the newest version. Also when I click on add integration there is no Skoda connect integration

**Comment by derjungewilde** [comment]
OK I had to completely uninstall - restart - install again and then I was able to see this integration in the UI. But If I want to configure it the VIN number was empty. I'm not able to select one there.

**Comment by Farfar** [comment]
> OK I had to completely uninstall - restart - install again and then I was able to see this integration in the UI. But If I want to configure it the VIN number was empty. I'm not able to select one there.

Is it completely empty?
![Screenshot_20210701-172317_Home Assistant~2](https://user-images.githubusercontent.com/1235384/124149842-43f25480-da91-11eb-819a-ace77d2176e3.jpg)
![Screenshot_20210701-172405_Home Assistant](https://user-images.githubusercontent.com/1235384/124149869-494f9f00-da91-11eb-98f0-8bbd9381d07a.jpg)

**Comment by derjungewilde** [comment]
> > OK I had to completely uninstall - restart - install again and then I was able to see this integration in the UI. But If I want to configure it the VIN number was empty. I'm not able to select one there.
> 
> Is it completely empty?
> ![Screenshot_20210701-172317_Home Assistant~2](https://user-images.githubusercontent.com/1235384/124149842-43f25480-da91-11eb-819a-ace77d2176e3.jpg)
> ![Screenshot_20210701-172405_Home Assistant](https://user-images.githubusercontent.com/1235384/124149869-494f9f00-da91-11eb-98f0-8bbd9381d07a.jpg)

Yes. If I click on VIN Number the dropdown ist empty. Nothing to select

**Comment by derjungewilde** [comment]
> > Yes. If I click on VIN Number the dropdown ist empty. Nothing to select
> 
> Very strange. And none of the versions after 1.0.40-RC2 was last working?
> If you check the Home Assistant log, what does the library report as version?
> 
> Edit:
> It should be a line like this:
> 2021-07-01 07:24:04 INFO (MainThread) [skodaconnect.connection] Init Skoda Connect library, version 1.0.42

there is no entry in the Supervisor and Core logs

**Comment by Farfar** [comment]
> > > Yes. If I click on VIN Number the dropdown ist empty. Nothing to select
> > 
> > 
> > Very strange. And none of the versions after 1.0.40-RC2 was last working?
> > If you check the Home Assistant log, what does the library report as version?
> > Edit:
> > It should be a line like this:
> > 2021-07-01 07:24:04 INFO (MainThread) [skodaconnect.connection] Init Skoda Connect library, version 1.0.42
> 
> there is no entry in the Supervisor and Core logs

Then the library hasn't been updated as it should. Don't know why, sorry. I'm not familiar with supervisor.
From library version 1.0.41 and onwards it logs the version number just for this specific reason.

**Comment by derjungewilde** [comment]
I tried to again uninstall - reboot - install latest version. But now the integration did not show up again in the guy. I tried several times. 

I will stick with the 1.0.40-RC2 for now

**Comment by Farfar** [comment]
> I tried again to uninstall - reboot - install latest version. But now the integration did not show up again in the guy.

It probably doesn't install correctly. Logs might tells you why.
There's not that much changed for the Octavia so you aren't missing out on other features than modifying departure schedule.

**Comment by Farfar** [comment]
To clarify. The issues here are related to yaml config not being imported to config flow and the library not being updated correctly.
New installations shouldn't be affected at all.
